charts/portal/templates/deployment-backend-administration.yaml

```diff
@@ -38,6 +38,12 @@ spec:
         app: {{ include "portal.fullname" . }}-{{ .Values.backend.administration.name }}
         {{- include "portal.selectorLabels" . | nindent 8 }}
     spec:
+      {{- with .Values.backend.administration.image.imagePullSecrets }}
+      imagePullSecrets:
+        {{- range . }}
+        - name: {{ .name }}
+        {{- end }}
```

Comment:
It would be great if you could change to a bit more simple retrieval, like [this one
](https://github.com/eclipse-tractusx/bpdm/blob/ccea3cb6c99a371c3d4f52153de4e42b045048aa/charts/bpdm/charts/bpdm-common/templates/_deployment.tpl#L45) overall